From 8116ea897a28cbe030f605ec3cc14128a5d0d957 Mon Sep 17 00:00:00 2001
From: HeYue <yue.he@intel.com>
Date: Fri, 9 Jun 2023 10:00:38 +0800
Subject: [PATCH] Fix the acrn sriov vulkan screen blur issue

Set the special bit to indicate the framebuffer.

Tracked-On: OAM-110165
Signed-off-by: HeYue <yue.he@intel.com>
---
 services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp b/services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp
index 8d685cfbdd7e..c9b41b361687 100644
--- a/services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp
+++ b/services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp
@@ -75,7 +75,8 @@ FramebufferSurface::FramebufferSurface(HWComposer& hwc, PhysicalDisplayId displa
     mConsumer->setConsumerName(mName);
     mConsumer->setConsumerUsageBits(GRALLOC_USAGE_HW_FB |
                                        GRALLOC_USAGE_HW_RENDER |
-                                       GRALLOC_USAGE_HW_COMPOSER);
+                                       GRALLOC_USAGE_HW_COMPOSER |
+                                       0x1000000U);
     const auto limitedSize = limitSize(size);
     mConsumer->setDefaultBufferSize(limitedSize.width, limitedSize.height);
     mConsumer->setMaxAcquiredBufferCount(
-- 
2.17.1

